app/controllers/web/console.php

```diff
@@ -1,8 +1,26 @@
 <?php
 
+use Appwrite\Utopia\Request;
 use Appwrite\Utopia\Response;
 use Utopia\App;
 
+App::init()
+    ->groups(['web'])
+    ->inject('request')
+    ->inject('response')
+    ->action(function (Request $request, Response $response) {
+        $time = (60 * 60 * 24 * 45); // 45 days cache
+
+        $response
+            ->addHeader('Cache-Control', 'public, max-age=' . $time)
+            ->addHeader('Expires', \date('D, d M Y H:i:s', \time() + $time) . ' GMT') // 45 days cache
```

Comment:
Can we not add cache here? The Service Worker in the console already handles that and I would like to avoid similiar issue we had with the old console

Reply:
Removed the cache headers 👍